nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenTCPRecord.java

```diff
@@ -393,6 +359,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                         // if discarding then bounce to the outer catch block which will close the connection and remove the flow file
                         // if keeping then null out the record to break out of the loop, which will transfer what we have and close the connection
                         try {
+                            getLogger().debug("Reading next record from recordReader");
```

Comment:
It seems like this debug line could generate a lot of output when enabled.  Since it does not provide any additional details, is it helpful, or should it be removed?